TranslatorReasonerAPI.yaml

```diff
@@ -579,3 +593,97 @@ components:
         URI.
       externalDocs:
         url: https://www.w3.org/TR/2010/NOTE-curie-20101216/
+    KnowledgeMap:
+      type: object
+      description: >-
+        Knowledge-map representation of this TRAPI web service.
+      properties:
+        nodes:
+          type: object
+          description: >-
+            Collection of node categories provided by this TRAPI web service,
+            indexed by Biolink class CURIEs.
+          additionalProperties:
+            $ref: '#/components/schemas/KMNode'
+        edges:
+          type: array
+          description: >-
+            List of edges/predicates provided by this TRAPI web service.
+          items:
+            $ref: '#/components/schemas/KMEdge'
+    KMNode:
+      type: object
+      description: >-
+        Description of a node category provided by this TRAPI web service.
+      properties:
+        id_prefixes:
+          type: array
+          description: >-
+            List of CURIE prefixes for the node category that this TRAPI web
+            service understands and accepts on the input.
+          items:
+            type: string
+          example: ['ChEMBL:', 'inchikey:']
+        count:
```

Comment:
I'm not sure that/how the Ranking Agent would use count information if it were provided, so I'm inclined to put it in the extended schema.

Reply:
done